subsys/bluetooth/host/conn.c

```diff
@@ -2904,6 +2904,25 @@ const bt_addr_le_t *bt_conn_get_dst(const struct bt_conn *conn)
 	return &conn->le.dst;
 }
 
+const bt_addr_t *bt_conn_get_dst_br(const struct bt_conn *conn)
+{
+#if defined(CONFIG_BT_CLASSIC)
+	if (conn == NULL) {
+		LOG_WRN("Invalid connect");
```

Comment:
```suggestion
		LOG_DBG("Invalid connect");
```

IMO we should not give a LOG_WRN when we are also returning an error code

Reply:
updated.